### PR TITLE
injecting CSS in document head with rollup-plugin-postcss

### PR DESCRIPTION
**Pull request by simonswiss**
Since I can't push to AWS, I have put the source client on `surge` to test in my demo. Works fine, the CSS is minified and injected in the document head.

I guess the `gulp-sass` task could be removed too - the sass is now handled via the postcss plugin.

More info on the plugin here: https://github.com/egoist/rollup-plugin-postcss

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <body>
     <div id="output"></div>
 
-    <script src="https://sdk.selzstatic.com/0.1.0/client.umd.polyfilled.js"></script>
+    <script src="https://selz-js-sdk-test.surge.sh/client.umd.polyfilled.js"></script>
```
Comment (sampotts) [inline docs/index.html]: Could you revert this one? Locally you can create `index-dev.html` and reference the files in `../dist/` directly 👍

File: docs/scripts.js [hunk 2]
```diff
@@ -90,23 +90,22 @@ document.addEventListener('DOMContentLoaded', () => {
     }
 
     const products = {
-        GBP: 'http://selz.co/1MaSYRU',
-        USD: 'http://selz.co/1rvbhT6',
+        USD: 'http://selz.co/EyQVaNOW4',
```
Comment (sampotts) [inline docs/scripts.js]: Could you restore those other two?

File: src/client.js [hunk 3]
```diff
@@ -23,19 +25,6 @@ class SelzClient {
             throw Error('User or domain are required');
         }
 
-        // Get the stylesheet
-        if (!document.getElementById('selz-client-styles')) {
-            fetch(config.urls.stylesheet())
```
Comment (sampotts) [inline src/client.js]: Can you remove the stylesheet reference in `config.urls`?

File: docs/index.html (hunk 1 above)
Comment (simonswiss) [reply docs/index.html]: oh, that's the use case for index-dev! 👍

Comment (sampotts) [reply docs/index.html]: Yeah I'll tidy it all up before we release it properly

File: src/client.js (hunk 3 above)
Comment (simonswiss) [reply src/client.js]: done

File: docs/scripts.js (hunk 2 above)
Comment (simonswiss) [reply docs/scripts.js]: done

**Comment by sampotts** [comment]
Cheers for doing this!

**Pull request merged**